.github/workflows/android-javadoc.yml

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2024 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+name: Javadoc
+
+on:
+  workflow_call:
+    inputs:
+      javadoc-build-path:
+        description: 'The build path where Javadoc files are stored (ex: code/edgeidentity/build/dokka/javadoc).'
+        required: true
+        type: string
+
+jobs:
+  javadoc:
+    name: Javadoc
+    runs-on: ubuntu-latest
+    # Skip the job if the input is an empty string
+    if: ${{ inputs.javadoc-build-path != '' }}
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4.2.2
+
+      - name: Setup Dependencies
+        id: setup-dependencies
+        uses: adobe/aepsdk-commons/.github/actions/android-setup-dependencies-action@gha-android-3.1.0
+
+      - name: Javadoc
+        run: make javadoc
+
+      - name: Set Javadoc File Name
+        run: |
+          FILE_TIMESTAMP=$(date -u +"%Y-%m-%d_%H-%M-%S")
+          echo "JAVADOC_FILE_NAME=javadoc-${FILE_TIMESTAMP}Z.log" >> $GITHUB_ENV
```

Comment:
Is there a way to add other information to the name such as version, branch, and/or repo name?

Reply:
Thanks for updating the filename! Small nitpick suggestions; swap branch name and version so version is just after the repo name, and maybe add a 'v' in front of the version while replacing invalid characters with '_' instead of blank (i.e. "v3_0_0" in your example)